data/json/items/armor/hoods.json

```diff
@@ -66,6 +67,32 @@
     "proportional": { "weight": 0.75, "volume": 0.75, "price": 0.5 },
     "flags": [ "VARSIZE", "STURDY", "HELMET_COMPAT", "UNDERSIZE" ]
   },
+  {
+    "id": "chainmail_junk_hood",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str": "chainmail coif" },
+    "description": "A customized chainmail hood.  Can be worn comfortably under helmets.",
+    "weight": "2006 g",
+    "volume": "500 ml",
+    "price": 5012,
+    "price_postapoc": 2500,
+    "to_hit": -1,
+    "material": [ "budget_steel" ],
+    "symbol": "[",
+    "looks_like": "balclava",
+    "color": "light_red",
+    "material_thickness": 4,
+    "flags": [ "VARSIZE", "STURDY", "HELMET_COMPAT", "OUTER" ],
+    "armor": [
+      {
+        "material": [ { "type": "budget_steel", "covered_by_mat": 95, "thickness": 1.2 } ],
+        "covers": [ "head" ],
+        "coverage": 95,
```

Comment:
this was 95 and 95 before, do you want it to go down to 92 (multiplicative), or 90 (addititive)?

Reply:
Covered by mat at 95, coverage at 100. That means it will always roll to protect with the 95% chance to actually protect. I'll fix it later tonight when I'm back home.

Reply:
What I'm saying tho is that should never happen. you should always need at least 1 material to have 100 proportional coverage. Otherwise in your example above it's saying this covers 100% of the limb but only protects 95%. So what is the other 5% being covered with?  It leads to weird stuff breaking like breathability but also means something can:
Get hit
Provide 0 protection
Take damage even though it didn't deflect any damage. In fact it's more likely to take damage since it didn't provide any damage reduction.